mlocal/frags/Makefile.stub

```diff
@@ -111,7 +121,7 @@ cniplugins:
 		cniplugin=$(cni_builddir)/$$name; \
 		if [ ! -f $$cniplugin ]; then \
 			echo " CNI PLUGIN" $$name; \
-			go build $(GO_BUILDMODE) $(GO_TAGS) $(GO_LDFLAGS) -o $$cniplugin $$p; \
+			go build $(GO_BUILDMODE) -tags "$(GO_TAGS)" $(GO_LDFLAGS) -o $$cniplugin $$p; \
```

Comment:
By removing `-tags` from the initial `GO_TAGS` definition in `mlocal/frags` we have to reference the flag here now, but we can add to `GO_TAGS` as we build up the correct set through the various `frags` included.